README.md

```diff
@@ -63,42 +60,59 @@ _How to get data from other services into RemNote._
+- [RemNote-Pomodoro](https://github.com/sirvan3tr/RemNote-Pomodoro) - Pomodoro timer and activity logger.
+- [remnote-schedule](https://github.com/hannesfrank/remnote-schedule) - Visualize your daily schedule. Inspired by [CrushEntropy.com](https://crushentropy.com/).
+- [English Dictionary](img/dictionary-plugin.png) - Look up definition of words
+  - Note: As RemNote Plugins are just webpages, you can use any webservice which works well in an iframe.
+
+<!-- TODO: Can we get a link (remnote.io/plugins/add?name=myplugin&width=) to install a plugin with default settings. This would be a first and simple to implement step to a plugin store.
+Martin said one can ask the devs to add a plugin.
+-->
 
-_RemNotes appearance can be customized on the `⮉ Custom CSS` page._
 
 ### Themes
-
-- **[RemNoteTheme](https://github.com/ethomasv/RemNoteTheme)** - De facto standard community theme supporting light and dark mode.
+Installed using [Stylus.](https://github.com/openstyles/stylus)
+- **[RemNoteTheme](https://github.com/ethomasv/RemNoteTheme)** - Modern design theme supporting light and dark mode.
 - [RemLight21](https://github.com/cannibalox/RemLight21) - A light theme for RemNote.
 
-### Mixins
-
+### Custom CSS Mixins
+_RemNote's appearance can be customized by pasting in and ticking css blocks on the `⮉ Custom CSS` page._
 _Not a full theme by itself, but styling specific features of RemNote. Can be used in conjunction with other themes._
 
 - [remnote-css-rem-types](https://github.com/hannesfrank/remnote-css-rem-types/) - Add icons for card types and practice direction.
 - [Fullscreen Queue](./css/README.md#fullscreen-queue) - Make queue larger on desktop.
 - [Highlight Focsed Rem](./css/README.md#highlight-focused-rem) - Highlight the rem you are editing.
 - [Scroll Further Down](./css/README.md#scroll-further-down) - Ability to scroll further down such that you don't have to write on the bottom of the page.
 
-☣️ Things that _will_ break at some point, ~~only use if you know what you are doing~~ DO NOT USE THIS ANYMORE. There is [a better way](https://forum.remnote.io/t/custom-rem-level-formatting-with-tags/168?u=hannesfrank):
 
-- ~~[Divider](./css/README.md#%EF%B8%8F-divider) - Format a rem as a divider, similar to notions `/divider`.~~
-- ~~[Comment](./css/README.md#%EF%B8%8F-comment)~~
+
+### Development
+
+_Things a plugin developer might need._
+- [Reference](docs/reference.md) in this repo - Undocumented links like an experimental trash to restore deleted rems and the list of Power-Up rems.
+- [Extensions, Plugins & Themes documenentation](https://www.remnote.io/documentation/nFTJ495uTxQTva9TL) in RemNote's Tutorial section.
+- [remnote-inspect](https://github.com/hannesfrank/remnote-inspect) - Show datastructure of context, focused Rem and Document.
+- [A feature request to allow more extensive formatting of rem.](https://forum.remnote.io/t/custom-rem-level-formatting-with-tags/168?u=hannesfrank)
```

Comment:
Haha, thanks. This is one of the things that I would not have thought of. But I can see that it is a starting point to get some inspiration.